staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -35,7 +35,7 @@ var (
 		&k8smetrics.HistogramOpts{
 			Name:    "rest_client_request_duration_seconds",
 			Help:    "Request latency in seconds. Broken down by verb and URL.",
-			Buckets: k8smetrics.ExponentialBuckets(0.001, 2, 10),
```

Comment:
can this be this way to calculate the calculate the SLO using 512 ms?

Reply:
Do you mean with the previous version? From what I can tell the previous version's biggest bucket is 512ms so it is possible. But I would argue that `k8smetrics.DefBuckets` gives more flexibility since it goes up to 10s with one of its buckets at 500ms.

Reply:
Yeah , I was trying to find a good answer for.current buckets ... But I can't defend it, the default ones are much better

Reply:
default buckets:
```
DefBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
```
for non long running requests, the timeout enforced on the server is `60s` (if client does not specify any timeout). Do you think it is worth adding more buckets on the upper bound?

This is how it's defined in the apiserver:
https://github.com/kubernetes/kubernetes/blob/1e6f3b5cd68049a3501782af8ff3ddd647d0b408/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L106-L107

Reply:
is going to make any difference to know that latency is 20 or 25 or 45 to the SRE/admin/operator? 
I don't think so, same as having such granularity on lower bounds, if there is nothing actionable I don't see the value of having too much information

Reply:
I agree with Antonio here, plus I don't think that from a client perspective we care about long-running requests. We essentially went to get insight on everything in between the client and the server, so `GET` requests are enough.

Reply:
i think it helps in debugging; during degradation, requests can take more than `10s`. Instead of seeing a flat line on 10s on the graph, it would be more revealing if we have a buckets for `{15, 20, 30, 45, 60}`, unless cardinality is a concern here.

Reply:
I'm going to use the same that the ones used in etcd then, that seems a common ground.
On the contrary to the apiserver, that  measure the latency to process a request and it maybe CPU only, this has always to measure the network roundtrip + the apiserver latency to process the request
```
[]float64{0.005, 0.025, 0.1, 0.25, 0.5, 1.0, 2.0, 4.0, 15.0, 30.0, 60.0},
```

Reply:
this keeps the same number of buckets and also is able to track latencies within the same limits the apiserver is doing